crash adding caches from list to GC bookmark list if debug logging is enabled

### Describe your problem!

Adding caches to bookmark lists crashs c:geo but only if debug logging is enabled. The caches are not added in this case. Without debug logging all is good.

```
2022-08-14 18:16:00.301 28874-19223/? E/cgeo: [network--5] [{"referenceCode":"GC9XY7N"}] {GCParser.addCachesToBookmarkList:1115/BookmarkUtils.lambda$processSelection$7:74/-$$Lambda$BookmarkUtils$_WgKmzaAXvMLK7o-vr3zH50lCWY.run:6/ScheduledDirectTask.call:38/ScheduledDirectTask.call:25/FutureTask.run:266/ThreadPoolExecutor.runWorker:1167/ThreadPoolExecutor$Worker.run:641}
2022-08-14 18:16:00.301 28874-19223/? W/System.err:     at cgeo.geocaching.utils.Log.e(Log.java:344)
2022-08-14 18:16:00.301 28874-19223/? W/System.err:     at cgeo.geocaching.connector.gc.GCParser.addCachesToBookmarkList(GCParser.java:1115)
2022-08-14 18:16:00.301 28874-19223/? W/System.err:     at cgeo.geocaching.connector.gc.BookmarkUtils.lambda$processSelection$7(BookmarkUtils.java:74)
2022-08-14 18:16:00.301 28874-19223/? W/System.err:     at cgeo.geocaching.connector.gc.-$$Lambda$BookmarkUtils$_WgKmzaAXvMLK7o-vr3zH50lCWY.run(Unknown Source:6)
2022-08-14 18:16:00.301 28874-19223/? W/cgeo: [network--5] UncaughtException {OOMDumpingUncaughtExceptionHandler.lambda$installUncaughtExceptionHandler$0:26/-$$Lambda$OOMDumpingUncaughtExceptionHandler$h5ON7aCd5MQnyQ5bQnIb7zK8lvY.uncaughtException:2/ThreadGroup.uncaughtException:1073/ThreadGroup.uncaughtException:1068/RxJavaPlugins.uncaught:434/RxJavaPlugins.onError:388/ScheduledDirectTask.call:45/ScheduledDirectTask.call:25}
    io.reactivex.rxjava3.exceptions.UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.lang.RuntimeException: Aborting on Log.e()
        at io.reactivex.rxjava3.plugins.RxJavaPlugins.onError(RxJavaPlugins.java:372)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:45)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
     Caused by: java.lang.RuntimeException: Aborting on Log.e()
        at cgeo.geocaching.utils.Log.e(Log.java:344)
        at cgeo.geocaching.connector.gc.GCParser.addCachesToBookmarkList(GCParser.java:1115)
        at cgeo.geocaching.connector.gc.BookmarkUtils.lambda$processSelection$7(BookmarkUtils.java:74)
        at cgeo.geocaching.connector.gc.-$$Lambda$BookmarkUtils$_WgKmzaAXvMLK7o-vr3zH50lCWY.run(Unknown Source:6)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:923) 
    
    
    --------- beginning of crash
2022-08-14 18:16:00.302 28874-19223/? E/AndroidRuntime: FATAL EXCEPTION: network--5
    Process: cgeo.geocaching, PID: 28874
    io.reactivex.rxjava3.exceptions.UndeliverableException: The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.lang.RuntimeException: Aborting on Log.e()
        at io.reactivex.rxjava3.plugins.RxJavaPlugins.onError(RxJavaPlugins.java:372)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:45)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
     Caused by: java.lang.RuntimeException: Aborting on Log.e()
        at cgeo.geocaching.utils.Log.e(Log.java:344)
        at cgeo.geocaching.connector.gc.GCParser.addCachesToBookmarkList(GCParser.java:1115)
        at cgeo.geocaching.connector.gc.BookmarkUtils.lambda$processSelection$7(BookmarkUtils.java:74)
        at cgeo.geocaching.connector.gc.-$$Lambda$BookmarkUtils$_WgKmzaAXvMLK7o-vr3zH50lCWY.run(Unknown Source:6)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
        at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:923) 
```

### How to reproduce?

1. Enable debug log in settings
2. open a list with some caches
3. menu > manage caches > add to bookmark list
4. select list

doesn't matter which list or caches are added

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Yes

### c:geo Version

master

### System information

_No response_

### Additional Information

_No response_

--- a/main/src/cgeo/geocaching/connector/gc/GCParser.java
+++ b/main/src/cgeo/geocaching/connector/gc/GCParser.java
@@ -1112,7 +1112,7 @@ public final class GCParser {
             }
         }
 
-        Log.e(arrayNode.toString());
+        Log.d(arrayNode.toString());
 
         try {
             Network.completeWithSuccess(Network.putJsonRequest("https://www.geocaching.com/api/proxy/web/v1/lists/" + listGuid + "/geocaches", arrayNode));

